app/views/layouts/_navbar.html.erb

```diff
@@ -292,29 +292,31 @@
 
             <!-- CART START -->
             <div class="dc-item">
+              <% if @item_orders %>
```

Comment:
There is no needs for this here, since `@item_orders` should be empty if there is no current order